src/libcommonserver/utility/utility_win.cpp

```diff
@@ -133,6 +133,13 @@ static bool init_private() {
 
     initTrusteeWithUserSID();
 
+    const char *getUseRightsFallbackMethod = std::getenv("KDRIVE_USE_GETRIGHTS_FALLBACK_METHOD");
```

Comment:
`CommonUtility::envVarValue` instead of `std::getenv`?

Reply:
Oups... Done